src/nls/it/strings.js

```diff
@@ -68,8 +68,8 @@ define({
     "ERROR_DELETING_FILE_TITLE"         : "Errore durante il tentativo di eliminazione del file",
     "ERROR_DELETING_FILE"               : "Errore durante il tentativo di eliminazione del file <span class='dialog-filename'>{0}</span>. {1}",
     "INVALID_FILENAME_TITLE"            : "Il nome del file non è valido",
-    "INVALID_FILENAME_MESSAGE"          : "Il nome del file non può contenere i seguenti caratteri: {0}",
-    "FILE_ALREADY_EXISTS"               : "Il file <span class='dialog-filename'>{0}</span> esiste già.",
+    "INVALID_FILENAME_MESSAGE"          : "Il nome {0} non può usare caratteri di sistema riservati, finire con un punto (.) o contenere i seguenti caratteri: <code class='emphasized'>{1}</code>",
```

Comment:
- `INVALID_FILENAME_MESSAGE` should become `I {0} non possono usare caratteri di sistema riservati, finire con un punto (.) o contenere i seguenti caratteri: <code class='emphasized'>{1}</code>`
- `DIRECTORY_NAMES_LEDE` should become `nomi delle cartelle`
- `FILENAMES_LEDE` should become `nomi dei file`